libcxx/CMakeLists.txt

```diff
@@ -177,6 +177,10 @@ set(LIBCXX_LIBDIR_SUFFIX "${LLVM_LIBDIR_SUFFIX}" CACHE STRING
     "Define suffix of library directory name (32/64)")
 option(LIBCXX_INSTALL_HEADERS "Install the libc++ headers." ON)
 option(LIBCXX_INSTALL_LIBRARY "Install the libc++ library." ON)
+cmake_dependent_option(LIBCXX_INSTALL_MODULES
```

Comment:
Can we install the modules if-and-only-if `LIBCXX_ENABLE_STD_MODULES` is `ON`? In other words, we might not need a separate option for installing the modules, I would think that enabling modules with `LIBCXX_ENABLE_STD_MODULES ` should take care of that.

Reply:
As discussed privately; this option currently needs a modern CMake version (3.26 or newer). We discussed the way to remove this dependency. So for now I keep it and work on that other solution first.

Reply:
What did 3.26 introduce that this relies on?

Reply:
I used the CMake module support to build the std module. This is used in the libc++ test suite.